File: rcmfadmin/Models/Grant.cs
namespace rcmfadmin.Models;
public class Grant : IDbItem<int>
{


  public int Id { get; set; }
  public int Amount { get; set; }
  public string Name { get; set; }
  public string Email { get; set; }
  public string Department { get; set; }
  public string Description { get; set; }
  public int Budget { get; set; } //needs to be a file?
  public string TrainingProvider { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
}

File: rcmfadmin.client/src/models/Grant.js
export class Grant {
  constructor(data) {
    this.id = data.id;
    this.amount = data.amount;
    this.name = data.name;
    this.email = data.email;
    this.department = data.department;
    this.description = data.description;
    this.budget = data.budget;
    this.trainingProvider = data.trainingProvider;
    this.createdAt = data.createdAt;
    this.updatedAt = data.updatedAt;
  }
}
